devices/tuya/_TZE200_t1blo2bj_neo_siren.json

```diff
@@ -0,0 +1,231 @@
+{
+  "schema": "devcap1.schema.json",
+  "uuid": "78685bfe-85cf-462a-bff9-059a876ffbb0",
+  "manufacturername": [
+    "_TZE200_t1blo2bj",
+    "_TZE204_t1blo2bj"
+  ],
+  "modelid": [
+    "TS0601",
+    "TS0601"
+  ],
+  "product": "Neo (Tuya) smart siren",
+  "sleeper": true,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_ON_OFF_LIGHT",
```

Comment:
The c++ code block tuya siren (they are not using the WD IAS cluster), not possible to use warning device type.

Reply:
I'm afraid I have to disagree here, at least partially. With a DDF supported device, the DDF defines what type of device it is, not the device itself anymore (meaning it's changable to your liking). So, if you make a warning device out of it, this generally allows following the expected code paths. Tuya is not blocked here per se.

In case the siren is still not able to sound up, it is more a matter of what you pass over for processing.

Reply:
I was not able to make it work as a warning device. Changing it to a light was a fix for me. More info at #6112